pkgs/applications/audio/baudline/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, fetchurl, libXmu, libXt, libX11, libXext, libXxf86vm, jack
+, makeWrapper
+}:
+
+let
+  rpath = stdenv.lib.makeSearchPath "lib"
```

Comment:
Why not `stdenv.lib.makeLibraryPath` instead? (That would also fix it on the closure-size branch.)